php/wipe_appwrite_bucket/index.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+require_once('vendor/autoload.php');
+
+use Appwrite\Client;
+use Appwrite\Services\Storage;
+
+return function ($req, $res) {
+    // Input validation
+    $bucketId = null;
+    try {
+        $payload = \json_decode($req['payload'], true);
+        $bucketId = \trim($payload['bucketId']);
+    } catch(\Exception $err) {
+        \var_dump($err);
+        $res->json([
+            'success' => false,
+            'message' => 'Payload is invalid.'
+        ]);
+        return;
+    }
+
+    if(empty($bucketId)) {
+        $res->json([
+            'success' => false,
+            'message' => 'Invalid bucket id.'
+        ]);
+        return;
+    }
+
+    // Make sure we have envirnment variables required to execute
+    if(!$req['variables']['APPWRITE_FUNCTION_ENDPOINT'] ||
+       !$req['variables']['APPWRITE_FUNCTION_PROJECT_ID'] ||
+       !$req['variables']['APPWRITE_FUNCTION_API_KEY']) {
+        $res->json([
+            'success' => false,
+            'message' => 'Please provide all required environment variables.'
+        ]);
+        return;
+    }
+
+    $client = new Client();
+    $storage = new Storage($client);
+
+    $client
+        ->setEndpoint($req['variables']['APPWRITE_FUNCTION_ENDPOINT'])
+        ->setProject($req['variables']['APPWRITE_FUNCTION_PROJECT_ID'])
+        ->setKey($req['variables']['APPWRITE_FUNCTION_API_KEY']);
+
+    $done = false;
+    while (!$done) {
+        $files = $storage->listFiles($bucketId)['files'];
+        foreach ($files as $file) {
+            $storage->deleteFile($bucketId, $file['$id']);
+        }
+
+        if (count($files) == 0) {
+            $done = true;
+        }
+    }
```

Comment:
Nice implementation! One thing, though, is if the bucket doesn't exist or if the API key is invalid, this function will error with an unhelpful error message. It would be good to wrap this in a try/catch and call:

```php
        $res->json([
            'success' => false,
            'message' => '<whatever error message>'
        ]);
```

Reply:
Added error handling, this is the message I get after passing a non-existing bucketId:
![screen-sel-230110-1949-06](https://user-images.githubusercontent.com/12299813/211625179-b50542ec-4a0d-4cb3-b7a5-568bf71c8eb4.png)
Payload: `{"bucketId":"asdf"}`